recipe/meta.yaml

```
@@ -1,26 +1,27 @@
 {% set name = "pytest-randomly" %}
-{% set version = "3.12.0" %}
+{% set version = "3.15.0" %}
+{% set name_url = name.split('-')[0]+"_"+name.split('-')[1] %}
 
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: d60c2db71ac319aee0fc6c4110a7597d611a8b94a5590918bfa8583f00caccb2
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name_url }}/{{ name_url }}-{{ version }}.tar.gz
+  sha256: b908529648667ba5e54723088edd6f82252f540cc340d748d1fa985539687047
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - pip
-    - python >=3.6
+    - python
+    - wheel
   run:
-    - python >=3.6
+    - python
     - importlib-metadata >=3.6.0
```

```suggestion
    - importlib-metadata >=3.6.0 # [py<310]
```
https://github.com/pytest-dev/pytest-randomly/blob/3.15.0/setup.cfg#L36

---

Thank you! I missed that.